cache.go

```diff
@@ -73,6 +76,7 @@ func (c *Cache) Store(t backend.Type, subtype string, id backend.ID) (io.WriteCl
 	return file, nil
 }
 
+// Load returns information from the cache.
 func (c *Cache) Load(t backend.Type, subtype string, id backend.ID) (io.ReadCloser, error) {
```

Comment:
What about Close()ing of the WriteCloser and ReadCloser that are returned in all those methods? Does the caller have to close them? Is that important? What happens if I don't do that? Maybe add a comment "It's the caller's responsibility to close the returned reader" or something like that (if it's important)?